README.md

```diff
@@ -30,6 +30,7 @@ probe is required.
 * openocd (version 0.11.0 or above)
 * screen
 * srecord
+* verilator (version 5.024-1 or above)
```

Comment:
I've been using 5.024, is there a difference between that and 5.024-1?

Reply:
I believe no difference. However, I saw it being reference [here](https://packages.ubuntu.com/source/oracular/verilator) as 5.024-1 unsure if this is an ubuntu thing?